exerciseTen.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
```

Comment:
I have noticed this is your other exercises as well. Your indentation is a bit all over the place. For now, copy and paste your html code into https://www.freeformatter.com/html-formatter.html and choose 4 spaces per level. I will explain you what this means next time but do bear in mind that indentation is really important